### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by omkarpawar2001** [aclap-dev/video-downloadhelper#594] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.hotstar.com/in/shows/man-dhaga-dhaga-jodte-nava/1260140962/leenas-plea-to-sarthak/1700036552/watch",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : error : unterminated entity reference hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/\r\nerror : unterminated entity reference hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/\r\nerror : unterminated entity reference hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/\r\nerror : unterminated entity reference hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/\r\nerror : unterminated entity reference hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/\r\nerror : unterminated entity reference hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/\r\nerror : unterminated entity reference hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/\r\nerror : unterminated entity reference hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/\r\n[dash @ 0000023834fbae80] Error when loading first fragment of playlist\r\n[in#0 @ 0000023834fbacc0] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://hses7-vod-cf-ace.cdn.hotstar.com/videos/hotstarint/1260140962/1700036552/v3/1716487996423/c811af9e895e40605b055c0c2c4227f1/master-937ba37513-1716491814501.mpd?ttl=86400&hdnea=exp=1716517102~acl=/videos/hotstarint/1260140962/1700036552/v3/1716487996423/c811af9e895e40605b055c0c2c4227f1~ttl=1800~type=paid~data=ip=kZf1BFOh9XJUcdup3AKDhG8e29VA1djGZDM7q3xAqwqfHz6CY41IjiN-userid=JcjVDk7dqTYGnFWpHV5rnJK56f6mDmEhkRGQPMhSDBkP-did=PfTBd1MT0XejJrN4JDog0PxaS7ry6Bwj5LIsiasFRHHw-cc=in-~hmac=fa0f1a8b7631b65667f842f43672584a85d75bcae4fbe61010e618a9ea51722e&Expires=1716517102&Signature=a6ttKuBK6k7RjVIzkBghMC8DCJf8dG7BRaDo09mKi2MBYVE8srWEemNNoEoamV6gQ21Q4oJ3UI1z9aafaSFgwEn6cwElhMh19a2qmjzcyqZqxxA65iA9nBXotTMBpOmxioS71JXpQUQrIZZ7iWAxEWknhmUU0WRcQofzuGNzjcNqRLM0aSL1Qk24qm5JtX6m2OLIDDvWdVyOpcGRbNN4g78G41VC3exYlwn8pOLq98AhBt7xcM3deIYIZPk5xdgAsmP6PUo5Ye9JOmwrsswhBHWM4qP67ElCeDvdYdjs-WTtSwO2oKZutHz09CUzS3SztCbFAwC7nanRm-EGabRFhQ__&Key-Pair-Id=APKAJC3ILMJXG4AINKJA.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "id": "coapp_failure_99ba4278-f6f5-423e-b1e5-82e99c6018a5",
    "downloadable_id": "downloadable_4895230451434752"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\pawar\\Downloads",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 9
}

**Comment by paulrouget** [comment]
Triage: ff-failure 403 + unterminated entity reference
